src/main/scala/io/scalac/tezos/translator/model/DTO.scala

```diff
@@ -0,0 +1,15 @@
+package io.scalac.tezos.translator.model
+
+import org.joda.time.DateTime
+
+object DTO {
```

Comment:
You can always pack it in package object. It is more like java naming convention. Classes inside of object have unneeded DTO suffix in my opinion. I think Errors in ErrorsDTO describes it's purpose and meaning quite well.  Treat this comment as minor one.